### PR TITLE
CU-860rwaj7v - fix component name of Shadcn element

### PR DESCRIPTION
**Pull request by vaionexbusra**
**TASK**
[Link](https://app.clickup.com/t/860rwaj7v)

**WHAT HAVE DONE**
import element files names fixed

### CODE REVIEW

File: src/components/UI/use-toast.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import * as React from 'react'
 const TOAST_LIMIT = 1
 const TOAST_REMOVE_DELAY = 1000000
 
+// eslint-disable-next-line no-unused-vars
```
Comment (paulsp94) [inline src/components/UI/use-toast.js:7]: leave a comment what it will be used for

File: src/firebase/auth.js [hunk 2]
```diff
@@ -174,6 +174,7 @@ export const signInWithEmail = async (formData, rememberMe) => {
 }
 export const resetUserPassword = async (email) => {
   try {
+    // eslint-disable-next-line no-unused-vars
```
Comment (paulsp94) [inline src/firebase/auth.js:177]: just remove the variable definition its useless

**Review by paulsp94 — CHANGES_REQUESTED** [review]

File: src/components/UI/use-toast.js (hunk 1 above)
Comment (vaionexbusra) [reply src/components/UI/use-toast.js]: checked after removing the code lines. It is working fine.

File: src/firebase/auth.js (hunk 2 above)
Comment (vaionexbusra) [reply src/firebase/auth.js]: removed the useless part. forgot password checked, working fine

**Review by paulsp94 — APPROVED** [review]
LGTM

**Pull request merged**